### PR TITLE
[MRG] Reduce linewidth of contours in plot_topomap

### PR DESCRIPTION
**Pull request by cbrnr**
The contours should not have the same width as the head geometry - using thinner lines separates the contours better from the head IMO.

Before:
topo_before

After:
topo_after

### CODE REVIEW

File: mne/viz/topomap.py [hunk 1]
```diff
@@ -669,7 +669,7 @@ def plot_topomap(data, pos, vmin=None, vmax=None, cmap=None, sensors=True,
         cont = None  # can't make contours for constant-valued functions
     else:
         cont = ax.contour(Xi, Yi, Zi, contours, colors='k',
-                          linewidths=linewidth)
+                          linewidths=linewidth / 2)
```
Comment (larsoner) [inline mne/viz/topomap.py]: better as `/ 2.` for Py2k

Comment (cbrnr) [reply mne/viz/topomap.py]: Don't we `from future import division` (or a similar thing in `six`)?

Comment (larsoner) [reply mne/viz/topomap.py]: No not in this file

https://github.com/cbrnr/mne-python/blob/34ba4de23ae6ff9a143acf8e9b6f64ab5701489b/mne/viz/topomap.py#L2

Comment (cbrnr) [reply mne/viz/topomap.py]: I see. But I guess it's done somewhere before importing this module, so it should work.

Anyway, I'll change it, but I'd rather add the `future` import above. If you insist, I'll put `2.`.

**Comment by agramfort** [comment]
please put the 2.

it's a good practice anyway for readability.

**Comment by cbrnr** [comment]
I disagree, I think `2.` reduces readability. But anyway, no problem!

**Comment by agramfort** [comment]
it makes it explicit that you work with floats

but anyway yes.

**Comment by cbrnr** [comment]
In Python 3 it doesn't matter if you divide floats or ints, the result will always be an int. I've updated it anyway. Pretty long discussion for such a tiny change 😄.

File: mne/viz/topomap.py (hunk 1 above)
Comment (larsoner) [reply mne/viz/topomap.py]: > I guess it's done somewhere before importing this module, so it should work.

Nope -- imports (including `__future__`) have local (file) scope only. Creating `untitled2.py` with:
```
from __future__ import division
my_number = 2
```
Then creating `untitled1.py` with:
```
from untitled2 import my_number
print(2 / my_number)
print(1 / my_number)
```
You get `1` then `0`.

Comment (cbrnr) [reply mne/viz/topomap.py]: Thanks for pointing this out! This is a nice example, although I would put it the other way round (import division in main, do the division in the module without the import). Same result though.

**Comment by agramfort** [comment]
thx @cbrnr 

just keep some love for our python 2 friends...

**Comment by larsoner** [comment]
Merging before the CIs come back happy, bold move @agramfort !

**Comment by larsoner** [comment]
I'm sure you ran unit test locally like in 2012 :)

**Comment by agramfort** [comment]
sure :)

**Comment by cbrnr** [comment]
Sorry, my Python 2 love is long gone - but I did it for MNE anyway 😆.

**Pull request merged**